diff --git a/utils/ipodpatcher/ipodio-posix.c b/utils/ipodpatcher/ipodio-posix.c
index cf1b3dd..1c27fce 100644
--- a/utils/ipodpatcher/ipodio-posix.c
+++ b/utils/ipodpatcher/ipodio-posix.c
@@ -290,18 +290,10 @@ cleanup:
 static int ipod_unmount(struct ipod_t* ipod)
 {
     char cmd[4096];
-    int res;
-
     sprintf(cmd, "/usr/sbin/diskutil unmount \"%ss2\"",ipod->diskname);
     fprintf(stderr,"[INFO] ");
-    res = system(cmd);
-
-    if (res==0) {
-        return 0;
-    } else {
-        perror("Unmount failed");
-        return -1;
-    }
+    system(cmd);
+    return 0;
 }
 #endif
 
diff --git a/utils/libtools.make b/utils/libtools.make
index 3ab150e..50a8191 100644
--- a/utils/libtools.make
+++ b/utils/libtools.make
@@ -168,7 +168,7 @@ $(TARGET_DIR)lib$(OUTPUT).a: $(LIBOBJS) $(addprefix $(OBJDIR),$(EXTRALIBOBJS))
 
 clean:
 	$(call rm, $(OBJS) $(OUTPUT) $(TARGET_DIR)lib$(OUTPUT)*.a $(OUTPUT).dmg)
-	$(call rm, $(OUTPUT)-* i386 ppc $(OBJDIR))
+	$(call rm, $(OUTPUT)-[0-9]* i386 ppc $(OBJDIR))
 
 # extra tools
 BIN2C = $(TOP)/tools/bin2c
